src/EventListener/ShippingExportEventListener.php

```diff
@@ -90,16 +80,21 @@ public function exportShipment(ExportShipmentEvent $exportShipmentEvent): void
         } catch (ByrdApiException $e) {
             $shippingExport->setState('failed');
             $this->entityManager->flush();
-
-            $exportShipmentEvent->addErrorFlash(
-                sprintf('Byrd error for order %s: %s', $order->getNumber(), $e->getMessage())
+            $this->requestStack->getSession()->getBag('flashes')->add(
```

Comment:
You could put the flashbag to a variable

Reply:
Fixed